### PR TITLE
api changes done according changes in all masterpages

### PR DESCRIPTION
**Pull request by HarshTatvasoft26**

### CODE REVIEW

File: APTITUDE/APTITUDETEST.Data/Data/DegreeRepository.cs [hunk 1]
```diff
@@ -40,6 +40,34 @@ public async Task<JsonResult> GetDegrees()
...
+                            case "desc":
+                                degreeData = degreeData.OrderByDescending(x => x.Name).ToList();
+                                break;
+                        }
+                        break;
+
+                    case "Level":
+                        switch (sortOrder)
+                        {
+                            case "asc":
```
Comment (ShubhamPadiyaTatvasoft) [inline APTITUDE/APTITUDETEST.Data/Data/DegreeRepository.cs:61]: change name to level

File: APTITUDE/AptitudeTest.API/Controllers/CollegesController.cs [hunk 2]
```diff
@@ -46,12 +46,14 @@ public async Task<JsonResult> Get(int id)
...
         /// <returns>filtered list of colleges</returns>
         [HttpGet]
-        public async Task<JsonResult> GetColleges(int? currentPageIndex, int? pageSize)
+        public async Task<JsonResult> GetColleges(int? currentPageIndex, int? pageSize, string? sortField, string? sortOrder)
         {
             CollegeQueryVM collegeQuery = new CollegeQueryVM()
             {
                 CurrentPageIndex = currentPageIndex,
-                PageSize = pageSize
+                PageSize = pageSize,
```
Comment (ShubhamPadiyaTatvasoft) [inline APTITUDE/AptitudeTest.API/Controllers/CollegesController.cs:54]: remove viewmodel pass all parameters

File: APTITUDE/APTITUDETEST.Data/Data/DegreeRepository.cs (hunk 1 above)
Comment (HarshTatvasoft26) [reply APTITUDE/APTITUDETEST.Data/Data/DegreeRepository.cs:61]: resolved

File: APTITUDE/AptitudeTest.API/Controllers/CollegesController.cs (hunk 2 above)
Comment (HarshTatvasoft26) [reply APTITUDE/AptitudeTest.API/Controllers/CollegesController.cs]: resolved

**Review by ShubhamPadiyaTatvasoft — APPROVED** [review]

**Pull request merged**